src/pages/webhooks/release-notes.md

```diff
@@ -9,6 +9,26 @@ keywords:
 
 These release notes describe the latest version of Adobe Commerce Webhooks.
 
+## Version 1.6.0
+
+### Release date
+
+October 30, 2024
+
+### Enhancements
+
+* Updated copyrights in the generated module files. <!--- CEXT-3508 -->
```

Comment:
List enhancements by impact. rather than Jira number. This one should be last. Adjust other enhancements here and in the events release notes as necessary, but I'm otherwise OK with the current orders.

Reply:
Made that adjustment and a small one for events